openhands_cli/stores/agent_store.py

```diff
@@ -86,9 +90,7 @@ def load(self, session_id: str | None = None) -> Agent | None:
                     if enabled_servers
                     else {},
                     "agent_context": agent_context,
-                    "condenser": agent.condenser.model_copy(update=condenser_updates)
-                    if agent.condenser
-                    else None,
+                    "condenser": condenser,
```

Comment:
@OpenHands can you help me think of a more fundamental approach to overwrite the source settings?